### PR TITLE
Copyright header & code style

### PR DESCRIPTION
**Pull request by pantor**
I've updated copyright headers and further code style, in particular for line length and comments.

### CODE REVIEW

File: src/Core/Evaluation.vala [hunk 1]
```diff
@@ -1,20 +1,21 @@
-/* Copyright 2014 Marvin Beckers <beckersmarvin@gmail.com>
```
Comment (danrabbit) [inline src/Core/Evaluation.vala]: Make sure to preserve Marvin's copyright here. It should read like:

```
Copyright (c) 2018 elementary LLC. (https://elementary.io)
              2014 Marvin Beckers <beckersmarvin@gmail.com>
```

File: src/Core/Scanner.vala [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-* Pantheon Calculator is distributed in the hope that it will be
-* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-* Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along
-* with Pantheon Calculator. If not, see http://www.gnu.org/licenses/.
-*/
+/*-
+ * Copyright (c) 2014-2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/Core/Scanner.vala]: Here also make sure to preserve Marvin's copyright

File: src/Core/Token.vala [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
-* Pantheon Calculator is distributed in the hope that it will be
-* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-* Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along
-* with Pantheon Calculator. If not, see http://www.gnu.org/licenses/.
-*/
+/*-
+ * Copyright (c) 2014-2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/Core/Token.vala]: And here :grimacing:

File: src/HistoryDialog.vala [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
-* Pantheon Calculator is distributed in the hope that it will be
-* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-* Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along
-* with Pantheon Calculator. If not, see http://www.gnu.org/licenses/.
-*/
+/*-
+ * Copyright (c) 2014-2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/HistoryDialog.vala]: And here too

File: src/MainWindow.vala [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
-* Pantheon Calculator is distributed in the hope that it will be
-* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-* Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along
-* with Pantheon Calculator. If not, see http://www.gnu.org/licenses/.
-*/
+/*-
+ * Copyright (c) 2014-2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/MainWindow.vala]: Another one :)

File: src/PantheonCalculator.vala [hunk 6]
```diff
@@ -1,20 +1,21 @@
...
-* Pantheon Calculator is distributed in the hope that it will be
-* useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-* Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along
-* with Pantheon Calculator. If not, see http://www.gnu.org/licenses/.
-*/
+/*-
+ * Copyright (c) 2014-2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/PantheonCalculator.vala]: And another one

**Comment by pantor** [comment]
Alright. I've also removed the authors file, as it looks very outdated in comparison to https://github.com/elementary/calculator/graphs/contributors.

**Pull request merged**